test/integration/kubernetes_deploy_test.rb

```diff
@@ -864,4 +864,15 @@ def test_cronjobs_can_be_deployed
     cronjobs = FixtureSetAssertions::CronJobs.new(@namespace)
     cronjobs.assert_cronjob_present("my-cronjob")
   end
+
```

Comment:
<img width="1552" alt="screen shot 2018-03-29 at 9 29 06 am" src="https://user-images.githubusercontent.com/12193286/38094599-d72a9fae-3333-11e8-97a4-38ea84bd4f73.png">

Not sure this is the best message, but I think we're getting to the tweak phase .

Reply:
Updated error:
<img width="1351" alt="screen shot 2018-04-02 at 10 57 21 am" src="https://user-images.githubusercontent.com/12193286/38203399-c7841d6c-3664-11e8-9020-d588910eb412.png">

I think we can skip the standard_timeout_message and stick with the info about increase --max-watch-seconds.